src/main/java/com/example/backendspringboottechiteasycontrollerles10/controllers/TelevisionController.java

```diff
@@ -0,0 +1,56 @@
+//package com.example.backendspringboottechiteasycontrollerles10.controllers;
+//
+//import com.example.backendspringboottechiteasycontrollerles10.exceptions.RecordNotFoundException;
+//import com.example.backendspringboottechiteasycontrollerles10.exceptions.TelevisionNameTooLongException;
+//import com.example.backendspringboottechiteasycontrollerles10.models.Television;
+//import org.springframework.http.HttpStatus;
+//import org.springframework.http.ResponseEntity;
+//import org.springframework.web.bind.annotation.*;
+//import java.util.ArrayList;
+//import java.util.List;
+//
+//@RestController
+//public class TelevisionController {
+//    private final List<Television> televisions;
+//    public TelevisionController() {
+//        televisions = new ArrayList<>();
+//        Television t = new Television();
+//        t.setName("Crystal");
+//        t.setBrand("Samsung");
+//        t.setPrice(549);
+//        t.setType("UHD 55AU7040");
+//        televisions.add(t);
+//    }
+//
+//    @GetMapping("/televisions")
+//    public ResponseEntity<List<Television>> getAllTelevisions() {
+//        return new ResponseEntity<>(televisions, HttpStatus.OK);
+////      return ResponseEntity.ok(televisions);
+//    }
+//
+//    @GetMapping("/televisions/{id}")
+//    public ResponseEntity<Object> getTelevision(@PathVariable("id") int id) {
+//        if (id >= 0 && id < televisions.size()) {
+//            Television t = televisions.get(id);
+//            return ResponseEntity.ok(t);
+//        } else {
+//            return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
+//        }
+//    }
+//
+//    @PostMapping("/televisions")
+//    public ResponseEntity<Television> createTelevision(@RequestBody Television newTelevision) {
+//        televisions.add(newTelevision);
+//        return new ResponseEntity<>(newTelevision, HttpStatus.CREATED);
```

Comment:
JE returned hier de Television die de gebruiker zojuist als RequestBody heeft opgegeven. De gebruiker weet al wat dat is, dus dat hoef je niet nog eens te returnen naar de gebruiker. Een message is hier voor nu voldoende.